README.md

```diff
@@ -0,0 +1,13 @@
+# CytoCherryPick
+
+## Welcome to CytoCherryPick
+CytoCherryPick is an open source software that provides a set of tools for the selection of representative images from a set of images.
+
+## Functions of CytoCherryPick
+* **Image Selection**: CytoCherryPick will select a representative image from a set of images.
+* **Image Distribution Analysis**: Choose your own representative image and CytoCherryPick will graph where in the image set distribution your image is located.
+
+### Install the CytoCherryPick to your environment
```

Comment:
Very optional: consider shortening this header. Sometimes the way people install Python work may not be into what they think of as an "environment". That said, simplification can help make the documentation here more approachable.
```suggestion
### Installation
```